### PR TITLE
ChangeLog for release v3.0.0 for real

### PR DESCRIPTION
**Pull request by cmcantalupo**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,49 @@
...
+- Official v3.0.0 release tag.
+- GEOPM Runtime support for non-MPI applications.
+- Integration with OpenPBS though plugins and launcher support.
+- Security improvements and bug fixes.
+- Additional GEOPM Service DBus APIs to support application profiling.
+- Communication between controller and application is managed by GEOPM Service.
+- Creation of topo-cache and responsibility for determining system topology is managed by GEOPM Service.
+- Update C++ standard requirement to C++17.
+- Add more signals and controls including GPU and platform features.
+- ConstConfigIOGroup for defining arbitrary constant settings/configurations as signals.
```
Comment (cmcantalupo) [inline ChangeLog]:
```suggestion
- ConstConfigIOGroup for defining constant settings/configurations as signals with json file.
```

File: ChangeLog [hunk 2]
```diff
@@ -1,3 +1,49 @@
+* Wed Oct 18 2023 Christopher M Cantalupo <christopher.m.cantalupo@intel.com> v3.0.0
+- Official v3.0.0 release tag.
+- GEOPM Runtime support for non-MPI applications.
+- Integration with OpenPBS though plugins and launcher support.
```
Comment (avilcheslopez) [inline ChangeLog]:
```suggestion
- Integration with OpenPBS through plugins and launcher support.
```

File: ChangeLog [hunk 3]
```diff
@@ -1,3 +1,49 @@
...
+- Add more signals and controls including GPU and platform features.
+- ConstConfigIOGroup for defining arbitrary constant settings/configurations as signals.
+- Increase the sample period of the monitor agent from 5 ms to 200 ms to reduce default CPU requirements of runtime.
+- Removal of libgeopmpolicy.so, use libgeopm.so instead.
+- Removal of geopmdpy.runtime module: no support for python based agents.
+- GEOPM_PERIOD / --geopm-period sets the sample period for controller in units of seconds.
+- GEOPM_INIT_CONTROL / --geopm-init-control to write a batch of controls at application startup.
+- GEOPM_CTL_LOCAL / --geopm-ctl-local disable controller's use of MPI.
+- GEOPM_PROGRAM_FILTER / --geopm-program-filter to select processes for profiling.
+- GEOPM_NUM_PROC sets number of process per node for controller process.
```
Comment (avilcheslopez) [inline ChangeLog]:
```suggestion
- GEOPM_NUM_PROC sets number of processes per node for controller process to track.
```

**Review by avilcheslopez — COMMENTED** [review]
I see there are some notes regarding changes related to MPI, however, I'm thinking it might be good to explicitly say where we stand on MPI support (to highlight what changed): MPI is no longer enabled by default (support still provided if needed).

**Review by avilcheslopez — APPROVED** [review]

**Pull request merged**